index.html

```diff
@@ -319,6 +321,18 @@ <h2>Glossary</h2>
 
 
 
+<a href="#links"><abbr title="Back to Alphabetical links">&#x2191;</abbr></a>
+
+<div class="letter_anchor" id="j">J</div>
+
+<p><dfn class="lint-ignore export">Jamo</dfn>. The basic unit used to form Hangul syllables, representing vowels and consonants in Korean.</p>
```

Comment:
I think you probably need to clarify that there are code points for jamos in Unicode, but that Korean is generally written using syllabic code points.  At the moment is sounds as if the default way to form Hangul syllables in encoding is to use jamos.  There's probably a Unicode FAQ page worth pointing to here.

Reply:
Fixed. Thanks.